Cargo.lock

```diff
@@ -287,6 +287,7 @@ dependencies = [
  "cargo",
  "cargo-util",
  "indexmap",
+ "itertools",
```

Comment:
I added `itertools` cause it greatly simplifies the `args` debug printing, and it's already used in the workspace.